docs/udf.rst

```diff
@@ -0,0 +1,179 @@
+.. _ksql-udfs:
+
+KSQL User Defined Function Reference
+====================================
+
+KSQL supports creating User Defined Functions (UDFs) via custom jars that are
+uploaded to the ext directory of the KSQL installation. At start up time KSQL scans the
```

Comment:
- Syntax nit: "ext directory" -> "``ext/`` directory"
- Also, we should clarify where the corresponding parent directory is in a Confluent Platform installation of KSQL (I wouldn't know it if you asked me).  That is, add sth like "by default, this would be the directory `/path/to/ksql/ext/`".